src/components/breakpointHandler.ts

```diff
@@ -0,0 +1,667 @@
+import * as vscode from 'vscode';
+import { DaCeVSCode } from '../extension';
+import { SdfgViewerProvider } from "./sdfgViewer"
```

Comment:
There are quite a few missing semicolons in this file, please make sure to add them - you can use the linter to highlight where they are missing